server/helpers/utils.js

```diff
@@ -1,17 +1,58 @@
 import slug from 'slug';
 import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+
+/**
+ * @param {string} password
+ * @return {string} hash
+ */
+export const hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
 
 /**
  * @param {string} title
  * @return {string} unique_slug
  */
-const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
+export const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
 
 /**
- * @param {string} password
- * @return {string} hash
+ * @param {object} req
+ * @return {object} formattedObject
  */
+export const toLowerCaseAndTrim = (req) => {
```

Comment:
I have given you feedback this already, `req` cannot be an argument here. Change to something with a much more generic name.